frontend/src/login.tsx

```diff
@@ -29,7 +29,8 @@ export function Login() {
     login(username, password).then(
       status => {
         if (status == 'logged_in') {
-          history.push("/")
+          //history.push("/")
```

Comment:
Better remove this comment before Martin sees it! :sweat_smile:

Reply:
Martin wanted the go back functionality, I do not understand :)

Reply:
Martin usually wants me to remove code in comments in my PRs. I don't mind too much though